decidim-core/db/migrate/20180305132906_rename_features_to_components.rb

```diff
@@ -9,5 +9,18 @@ def change
     if index_name_exists?(:decidim_components, "index_decidim_features_on_decidim_participatory_space")
       rename_index :decidim_components, "index_decidim_features_on_decidim_participatory_space", "index_decidim_components_on_decidim_participatory_space"
     end
+
+    # rubocop:disable Rails/SkipsModelValidations
+    PaperTrail::Version.where(item_type: "Decidim::Feature").update_all(item_type: "Decidim::Component")
+    Decidim::ActionLog.where(resource_type: "Decidim::Feature").update_all(resource_type: "Decidim::Component")
```

Comment:
You need to create a custom class for these models, otherwise the migration will fail

```ruby
  class ActionLog < ApplicationRecord
    self.table_name = :decidim_action_logs
  end

  class Version < ApplicationRecord
    self.table_name = :versions
  end
```